qa/tasks/scripts/ses_replace_disk_part1.sh

```diff
@@ -0,0 +1,28 @@
+set -ex
+
+declare -a random_minion_fqdn="$@"
+
+echo "### Getting random minion and its random OSD ###"
+random_minion=`echo $random_minion_fqdn | cut -d . -f 1`
+random_osd=`ceph osd tree | grep -A 1 $random_minion | grep -o "osd\.".* | awk '{print$1}'`
+osd_id=`echo $random_osd | cut -d . -f 2`
+
+vg_name=`salt $random_minion_fqdn cmd.run "find /var/lib/ceph/osd/ceph-$osd_id -type l -name block -exec readlink {} \; | rev | cut -d / -f 2 | rev" | tail -1 | tr -d ' '`
```

Comment:
Here and after could you please use `$()` instead backticks ``. And split the command in several lines.